Update ZBT-1 index.html

Hedda opened the pull request:
For reference, [EmberZNet (a.k.a. Ember adapter) support in Zigbee2MQTT](https://www.zigbee2mqtt.io/guide/adapters/emberznet.html)  is no longer experimental so that is stable now in Z2M. 

* https://www.home-assistant.io/connectzbt1/

<img width="309" height="103" alt="image" src="https://github.com/user-attachments/assets/ac33ad11-7017-42a1-a347-2d093f41b7f9" />

<img width="408" height="42" alt="image" src="https://github.com/user-attachments/assets/06aeb636-f2e2-423e-95f5-5e5facaffe73" />

I would however like to question though if the mentioning of Zigbee2MQTT should not just be comlely removed here? Why specifically mention Zigbee2MQTT when Home Assistant have its own built-in native [ZHA integration component](https://www.home-assistant.io/integrations/zha) for Zigbee? As I understand it Zigbee2MQTT actually already have a larger userbase than ZHA today:

* https://github.com/home-assistant/home-assistant.io/issues/41333

Why not take this chance to further promote ZHA as the official integration and hopefully make it attract more users which in turn could attract more developers to Home Assistant and ZHA (instead of redirecting them a third-party standalone project)? 

* https://www.home-assistant.io/integrations/zha

If the ZHA integration and the zigpy libraries it depends on (which is owned and governed by the Open Home Foundation) had both more users to attract more developers then maybe it could be improved to some day become as good of an experince as Zigbee2MQTT has at being the best-in-class Zigbee Gateway for Home Assistant, even for existing Z2M advanced users which most of today say Zigbee2MQTT is better than ZHA. This should hopefully be a long-term challage for the roadmap regardless.

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Anyway, there is no need to state that [EmberZNet (a.k.a. Ember adapter) support Zigbee is experimental as it has been stabalized.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Hedda committed:
Update ZBT-1 index.html

EmberZNet ("Ember") support in Zigbee2MQTT is no longer experimental.

I like to question though if mention of Zigbee2MQTT should not just be removed?

frenck commented on source/connectzbt1/index.html:
```diff
@@ -112,7 +112,7 @@
       <span>Compatible</span>
       <span>Zigbee</span>
-      <span>Works with Home Assistant and Zigbee2MQTT<sup>1</sup></span>
+      <span>Works with Home Assistant's ZHA integration or Zigbee2MQTT</span>
```
I don't think it is matter that we list the ZHA integration. Besides that, we don't call it ZHA anyways.

../Frenck

Hedda replied:
Would it sound better "Works with Home Assistant's own Zigbee integration or Zigbee2MQTT"?

I think the most important word here though is "or" which will replace the word "and" there.

That is, fact is that one single ZBT-1 adapter can not be used with ZHA "and" Zigbee2MQTT.

frenck replied:
I think it is fine the way it is. It is the build-in.

Let's revert this bit.

../Frenck

Hedda replied:
Do you mean

"Works with Home Assistant or Zigbee2MQTT"?

or 

"Works with Home Assistant's built-in Zigbee gateway or Zigbee2MQTT"?

or 

"Works with Home Assistant built-in or Zigbee2MQTT"?

or 

"Works with Home Assistant's Zigbee integration or Zigbee2MQTT"?

frenck replied:
I mean reverting the change to what it was.

../Frenck

Hedda replied:
Not even from "Works with Home Assistant **and** Zigbee2MQTT" to "Works with Home Assistant **or** Zigbee2MQTT"?

Agaim, my main point is that a single ZBT-1 adapter can not be used with ZHA "**and**" Zigbee2MQTT.

That is, if  buy one ZBT-1 then you can connect it to  Home Assistant **or** Zigbee2MQTT, not both at the same time.

IMHO I therefore think it the important wording here is that "or" should replace the word "and" there.

frenck replied:
Sure; that is fine; both could work though

Hedda replied:
Removed reference to ZHA integration as requested by frenck.

It now says "Works with Home Assistant or Zigbee2MQTT"

Hedda committed:
Update compatibility description in index.html

Removed reference to ZHA integration as requested by frenck.

It now says "Works with Home Assistant or Zigbee2MQTT"

Hedda committed:
Merge branch 'current' into patch-824360

Hedda commented:
Sorry, I just forgot to press the "Ready for review" button

frenck reviewed (approved):
Thanks, @Hedda 👍 

../Frenck
